tests/use_cases/test_request_email_address_change.py

```diff
@@ -86,7 +86,7 @@ def test_that_requests_are_rejected_if_new_email_address_is_already_taken_by_mem
             ("@b",),
         ]
     )
-    def test_that_email_addresses_that_are_definityly_not_valid_are_rejected_as_new_addresses(
+    def test_that_email_addresses_that_are_definitly_not_valid_are_rejected_as_new_addresses(
```

Comment:
Thanks, I amended the change to the commit. I'll wait with merging until @seppeljordan has reviewed it, because this PR involves a server-side change.